templates/docker-compose/Makefile

```diff
@@ -1,4 +1,5 @@
 SELF_FILENAME := $(word $(words $(MAKEFILE_LIST)),$(MAKEFILE_LIST))
+TIMESTAMP := $$(date +%s)
```

Comment:
What does this produce? `Y-m-d` kind of format would be good.

Reply:
It creates a timestamp e.g. `1618490824`

Reply:
The main reason we have this is to eliminate backups overwriting each other. I'll see a friendlier format

Reply:
```
$ date +%s-%F
1618495023-2021-04-15
```

Above might be a good format? Easy to sort / manipulate with the timestamp prefix.

Reply:
don't have a strong opinion on this. I'll use `date +%s-%F`